doc/source/Linux.rst

```diff
@@ -0,0 +1,190 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
```

Comment:
`demonstration on suite` seems like unusual english, what are you trying to express here?  `some demonstration of several of the plugins available from`?

Also, this sentence just stops without saying where from...  5;)